app/Jobs/PlatformStatsSummaryJob.php

```diff
@@ -163,7 +181,54 @@ public function handle( DatabaseManager $manager ): void
         if( !App::runningUnitTests() ) {
             print( json_encode($summary) . PHP_EOL );
         }
+    }
+
+    private function fetchPagesInNamespace(string $wikiDomain, int $namespace): array
```

Comment:
If I understand it correctly, this method is copied verbatim from the `RebuildQueryserviceData` command. To remove the duplication, what do you think about doing the following:

- create a `PageFetcher` trait in `App\Traits` that contains this method as well as the `apiUrl` property.
- `use` this trait here as well as in `RebuildQueryserviceData`

Reply:
done